content/resources/whitepaper.md

```diff
@@ -17,7 +17,7 @@ Last updated: 2022-01-25
 
 ## Summary
 
-OmniBOR is an application of the [git](https://en.wikipedia.org/wiki/Git) [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph), a widely used merkle tree with a flat-file storage format, to the challenge of creating build artifact dependency graphs in today's language-heterogeneous open source environments. Contrary to the name's appearance, OmniBOR is neither dependent on `git` nor is it a Software Bill Of Materials (SBOM).
```

Comment:
Thanks for clipping this, it should have been gone with the rename from GitBOM -> OmniBOR :)